cloudferrylib/os/network/network_utils.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2016 Mirantis Inc.
```

Comment:
I've separated it because we use the prepare_networks function in the PrepareNetworks action (cloudferrylib/os/actions/prepare_networks.py) and when we trying to recreate ports in the TransportInstance action (cloudferrylib/os/actions/transport_instance.py).